index.html

```diff
@@ -0,0 +1,112 @@
+        <!------------------ HOME ------------------>
+        <section class="home">
+            <div class="presentation">
+                <h1>"Hello! Armys"</h1>
+                <h2>봄날 WORLD TOUR 2025</h2>
+                <p class="text">Armys, are you ready to get hyped up ? </br>BTS are finally back to the stage to perform as a group, after a two-year hiatus. Spring Day 2025, the most awaited Wolrd Tour, is going to take them to 15 cities across Asia, Africa and Europe starting from 1st July. </br>Get your army bombs ready, many surprises are waiting for you at concert venues</p>
+                <h3>2025년7월3일 ~ 8월2일</h3>
+                <p class="venue">@SEOUL Olympic Stadium, TOKYO Dome, Prince Moulay Abdellah Stadium and more ~ </p>
+            </div>
+        </section> 
+
+        <section class="schedule">
+            <h2>Main program</h3>
+            <span class="line-break"></span>
+            <div class="program-container">
+                <div class="program">
+                  <img class="img1" src="./assets/img/program3.png" alt="">
+                  <h3>Press Conference</h3>
+                  <p>The members will answer on press questions and talk about their journey as solo artists before army enlistment</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program2.png" alt="">
+                  <h3>Concert Day 1</h3>
+                  <p>Voice check starts at 9 am. The concert takes place from 7 to 10pm. Songs from HYYH, Wings and Love Yourself alumbs will be performed</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program2.png" alt="">
+                  <h3>Concerts Day 2</h3>
+                  <p>Voice check starts at 9 am. The concert takes place from 7 to 10pm. Songs from Map of The Soul, Proof and solo albums will be performed</p>
+                </div>
+                <div class="program">
+                  <img class="img2" src="./assets/img/Program5.png" alt="">
+                  <h3>Fan Meeting</h3>
+                  <p>The members will join their fans in an exlusive meeting to sign albums and have fun together with Armys</p>
+                </div>
+                <div class="program">
+                  <img class="img1" src="./assets/img/Program1.png" alt="">
+                  <h3>Show Time</h3>
+                  <p>Show of BTS three movies: Burn The Stage, Bring The Soul and Break The Silence</p>
+                </div>
+              </div>
+              <a href="#">Get your tickets now on Weverse Shop</a>
+        </section>
+        <!------------------ GROUP MEMEBERS SECTION ------------------>
+        <section class="global-section">
+            <h3>Group Members</h3>
+            <hr>
+            <section class="gm-section">
+                <!--- Dynamically generated with javascript --->
+            </section>
+        </section>  
```

Comment:
- [ ] Good job so far 👍  But I notice that your speaker section is different from the original design 👍 according to the project requirement this section contains six individual articles and this section's articles are organized in two columns of three articles each. And please make sure to look like [ the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please go through it and make sure to use the exact same colors, for text, backgrounds, borders, etc, from the original design as required for this project